Application/EdFi.Ods.Admin/Initialization/InitializationEngine.cs

```diff
@@ -79,23 +80,23 @@ public void CreateUsers()
             }
         }
 
-        private void CreateUser(UserInitializationModel user)
+        private async void CreateUser(UserInitializationModel user)
         {
             WebSecurity.CreateUserAndAccount(
                 user.UserName,
                 user.Password,
-                new
-                {
-                    FullName = user.Name
-                });
+                new {FullName = user.Name});
 
             foreach (var role in user.Roles)
             {
                 Roles.AddUserToRole(user.Email, role);
             }
 
             WebSecurityService.UpdatePasswordAndActivate(user.UserName, user.Password);
-            _clientAppRepo.SetDefaultVendorOnUserFromEmailAndName(user.Email, user.Name);
+
+            var namespacePrefixes = user.NamespacePrefixes.Select(nsp => nsp.Name).ToList();
+
+            await _clientAppRepo.SetDefaultVendorOnUserFromEmailAndNameAsync(user.Email, user.UserName, namespacePrefixes);
```

Comment:
If we do use async, we need to be configure awaiting false everywhere, this is a library so we don't want to mess with SynchronizationContext.

Reply:
done